messages/php/Makefile

```diff
@@ -4,19 +4,20 @@ JSONSCHEMAS = $(shell find ../jsonschema -name "*.json")
 
 clean: clean-build
 
-.codegen: build/messages.php
+.codegen: $(JSONSCHEMAS) ../jsonschema/scripts/codegen.rb ../jsonschema/scripts/templates/php.php.erb
+	ruby ../jsonschema/scripts/codegen.rb Php ../jsonschema php.php.erb > build/messages.php
+	ruby ../jsonschema/scripts/codegen.rb Php ../jsonschema php.enum.php.erb >> build/messages.php
+	csplit --quiet --prefix=build/Generated --suffix-format=%02d.php.tmp --elide-empty-files build/messages.php /^.*[.]php$$/ {*}
+	rm build/messages.php
+	rm -rf src-generated/*
+	for file in build/Generated**; do mkdir -p src-generated/$$(head -n 1 $$file | sed 's/[^/]*.php$$//'); done
```

Comment:
This is required to make the subfolders - the sed ends up trying to make `src-generated/` a bunch of times but I think it's harmless